### ISSUE TITLE
change the dataset

### ISSUE THREAD

**Issue by iamzhangyunlu** [cvqluu/Additive-Margin-Softmax-Loss-Pytorch#4]
Hi there, thank you for your code. Here is a question: I changed the dataset using **FER2013** instead of Fashion MNIST, the original fer2013 is csv format, and I changed it into jpg, and the dimension become 3 (original dimension is 1, grey images).

Will that affect the 3D mapping pics? Cuz I achieved bad results by changing the dataset. @cvqluu
